dune/grid/cpgrid/GridHelpers.hpp

```diff
@@ -399,6 +399,74 @@ beginCellCentroids(const Dune::CpGrid& grid);
 double cellCentroidCoordinate(const Dune::CpGrid& grid, int cell_index,
                               int coordinate);
 
+/// \brief Get the centroid of a cell.
+/// \param grid The grid whose cell centroid we query.
+/// \param cell_index The index of the corresponding cell.
+const double* cellCentroid(const Dune::CpGrid& grid, int cell_index);
+
+/// \brief Get the volume of a cell.
+/// \param grid The grid the cell belongs to.
+/// \param cell_index The index of the cell.
+double cellVolume(const  Dune::CpGrid& grid, int cell_index);
+
+/// \brief An iterator over the cell volumes.
+class CellVolumeIterator
+    : public Dune::RandomAccessIteratorFacade<CellVolumeIterator, double, double, int>
```

Comment:
This (public inheritance from `Dune::RandomAccessIteratorFacade<>`) appears to rely on namespace pollution from opm-autodiff's [`opm/autodiff/GridHelpers.hpp`](https://github.com/blattms/opm-autodiff/blob/uggridhelpers-completion/opm/autodiff/GridHelpers.hpp#L81).  Any chance you could
```C++
#include <dune/common/iteratorfacades.hh>
```
here (in dune-cornerpoint's edition of `GridHelpers.hpp`, [`dune/grid/cpgrid/GridHelpers.hpp`](https://github.com/blattms/dune-cornerpoint/blob/uggridhelpers-completion/dune/grid/cpgrid/GridHelpers.hpp)) rather than there?